src/SignalR/common/Http.Connections/src/Microsoft.AspNetCore.Http.Connections.csproj

```diff
@@ -42,8 +42,4 @@
     <InternalsVisibleTo Include="Microsoft.AspNetCore.Http.Connections.Tests" />
   </ItemGroup>
 
-  <ItemGroup>
```

Comment:
This is a shared framework only project, shouldn't have been claiming to support the browser.